cmd/dm-ctl/main.go

```diff
@@ -63,6 +66,40 @@ func main() {
 		os.Exit(2)
 	}
 
+	if len(cmdArgs) > 0 {
+		commandMode(cmdArgs)
+	} else {
+		interactionMode()
+	}
+}
+
+func collectArgs(args []string) []string {
+	collectedArgs := make([]string, 0, len(args))
+	for i := 0; i < len(args); i++ {
+		switch strings.ToLower(args[i]) {
+		case "start-task", "query-status", "stop-task", "pause-task", "resume-task", "show-ddl-locks", "unlock-ddl-lock", "break-ddl-lock":
+			{
+				collectedArgs = append(collectedArgs, args[i:]...)
```

Comment:
should we limit the format of the command?
like `./dmctl --master-addr=127.0.0.1:8261 start-task task.yaml`
or `./dmctl balabala hahaha start-task aha task.yaml`

Reply:
Yes, after I communicate with @ericsyh , we both think we should limit the format of command:
1. one command one task operation.
2. put task operation at last.